rotkehlchen/tests/api/test_ethereum_transactions.py

```diff
@@ -1505,6 +1506,7 @@ def test_no_value_eth_transfer(rotkehlchen_api_server: 'APIServer'):
     assert result['entries'][0]['decoded_events'][0]['entry']['balance']['amount'] == '0'
 
 
+@pytest.mark.parametrize('ethereum_accounts', [[TEST_ADDR1, TEST_ADDR2]])
```

Comment:
why not freeze time here and in 1559 too?

Reply:
it was not needed as the queries didn't depend on the time

Reply:
Thing is a lot of things underneat may depend on time and all the queries that may be formed and recorded may be time dependent.

Result may not, but the queries sent may be. I guess if not needed it's fine but we also gotta think about maintenance and how tests will change in the future.